runtime/js/99_main.js

```diff
@@ -455,10 +455,35 @@ delete Object.prototype.__proto__;
 
   let hasBootstrapped = false;
 
+  // A helper function that will bind our own console implementation
+  // with default implementation of Console from V8. This will cause
+  // console messages to be piped to inspector console.
+  //
+  // We are using `Deno.core.callConsole` binding to preserve proper stack
+  // frames in inspector console.
```

Comment:
```suggestion
  // We are using `Deno.core.callConsole` binding to preserve proper stack
  // frames in inspector console. This has to be done because V8 considers
  // the last JS stack frame as gospel for the inspector. In our case we
  // specifically want the latest user stack frame to be the one that matters
  // though.
```

Let's add some more docs on why this is needed.